ts_scripts/modelarchiver_utils.py

```diff
@@ -1,40 +1,32 @@
 import os
 import sys
 
-
 REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")
 sys.path.append(REPO_ROOT)
 
 
 def test_modelarchiver():
     os.chdir("model-archiver")
 
-    # Lint test
-    print("## Started model archiver linting")
-    ma_dir = os.path.join("model_archiver", ".")
-    rc_file_path = os.path.join(".", "model_archiver", "tests", "pylintrc")
-    py_lint_cmd = f"pylint -rn --rcfile={rc_file_path} {ma_dir}"
-    print(f"## In directory: {os.getcwd()} | Executing command: {py_lint_cmd}")
-    py_lint_exit_code = os.system(py_lint_cmd)
-
     # Execute python unit tests
     print("## Started model archiver pytests - unit tests")
     ut_dir = os.path.join("model_archiver", "tests", "unit_tests")
     coverage_dir = os.path.join(".")
-    results_dir_name = "result_units"
-    py_units_cmd = f"python -m pytest --cov-report html:{results_dir_name} --cov={coverage_dir} {ut_dir}"
+    report_output_dir = os.path.join(ut_dir, "coverage.xml")
+
+    py_units_cmd = f"python -m pytest --cov-report xml:{report_output_dir} --cov={coverage_dir} {ut_dir}"
```

Comment:
Why change from html to xml?

Reply:
Tools like `codevov` need an `xml`. `html` while more human readable is not useful in this scenario because it's going to be on a CI machine